docs/introduction/what-is-debug-flow.md

```diff
@@ -2,12 +2,26 @@
 
 <!--@include: ../../README.md#introduction-->
 
+This is how the application looks like from a conceptual point of view.
```

Comment:
nit: maybe rephrase

Conceptual wise, the application serves as a backend server running in the repository, which:
- serves the frontend application
- interacts with the git version control
- manages the state of the flows stored there
<PIC>